### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by patrickimran**
Removes noisy print statements from unit_benchmark, and fixes a bug in RegressionTask where the sample_ids were saved in the y_true column of the results table.
Only affects the y_true column in the results table, as the list `y_test` was used directly in any calculations (RMSE). These true values correspond to the value of target variable for each sample_ID in the original dataset, so they are recoverable if we really need them.

### CODE REVIEW

File: q2_mlab/learningtask.py [hunk 1]
```diff
@@ -284,7 +284,7 @@ def cv_fold(self, train_index, test_index):
         self.results["RUNTIME"][curr_indices] = runtime
         self.results["CV_IDX"][curr_indices] = self.cv_idx
         self.results["Y_PRED"][curr_indices] = y_pred
-        self.results["Y_TRUE"][curr_indices] = y_test_ids
+        self.results["Y_TRUE"][curr_indices] = y_test
```
Comment (ElDeveloper) [inline q2_mlab/learningtask.py:363]: Can you add a test for this value?

**Review by ElDeveloper — CHANGES_REQUESTED** [review]

File: q2_mlab/learningtask.py (hunk 1 above)
Comment (patrickimran) [reply q2_mlab/learningtask.py:363]: Thanks, just added in a test for these!

File: q2_mlab/tests/test_unit_benchmark.py [hunk 2]
```diff
@@ -188,7 +189,54 @@ def testReturnBestModel(self):
         )
 
         self.assertTrue(best_model)
-        self.assertEquals(results['ACCURACY'].max(), best_accuracy)
+        self.assertEqual(results['ACCURACY'].max(), best_accuracy)
+
+    def testResultTable(self):
```
Comment (ElDeveloper) [inline q2_mlab/tests/test_unit_benchmark.py]:
```suggestion
    def test_result_table(self):
```

File: q2_mlab/tests/test_unit_benchmark.py [hunk 3]
```diff
@@ -188,7 +189,54 @@ def testReturnBestModel(self):
...
+
+        # Test Regression table
+        results, _, _ = q2_mlab._unit_benchmark(
+            table=test_table,
+            metadata=continuous_target,
+            algorithm="LinearSVR",
+            params=self.svr_params,
+            n_repeats=1
+        )
+        for sampleid in results.SAMPLE_ID:
```
Comment (ElDeveloper) [inline q2_mlab/tests/test_unit_benchmark.py:222]: I would recommend using Pandas' utilities for checking that two dataframes are almost equal:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html

**Review by ElDeveloper — CHANGES_REQUESTED** [review]

**Comment by ElDeveloper** [comment]
Thanks @patrickimran

File: q2_mlab/tests/test_unit_benchmark.py (hunk 3 above)
Comment (gwarmstrong) [reply q2_mlab/tests/test_unit_benchmark.py:222]: Agreed.

**Pull request merged**